requirements/requirements.txt

```diff
@@ -5,5 +5,5 @@ dask>=2023.1.0
 numba==0.56.4
 multiprocess==0.70.14
 wget==3.2
-PyYAML==6.0.2
+PyYAML==6.0
```

Comment:
I think `6.0` has some crucial bug that prevent installation .. was there a reason for reverting from `6.0.2`?

Reply:
No specific reason, I just adapted when installing new and then using `pip -list`. Will try with the old version